pkg/platform/packet/packet_test.go

```diff
@@ -355,3 +355,47 @@ func TestTerraformAddDeps(t *testing.T) {
 		})
 	}
 }
+
+func Test_areUniqueStrings(t *testing.T) {
```

Comment:
We shouldn't test private method directly. We should rather validate certain configuration cases.

Reply:
Could you elaborate what does ` shouldn't test private method directly` mean? What's wrong with testing the private method?

Reply:
Testing private methods test implementation directly, which creates [fragile tests](https://ordina-jworks.github.io/testing/2019/08/23/test-design.html), which require changing/fixing when you do the refactoring.

Tests should define specification, so testing public API of your code, which is part of the contract between other elements in the software.

Reply:
I wouldn't be so strict with this kind of "rules". I think a unit test for this method makes sense: it's a well defined function that's unlikely to change with any refactoring. (See our other comments on separating checking and making the list unique though).